### PR TITLE
changed user model, package.json, and login

### PR DESCRIPTION
**Pull request by juanmartinez238**

### CODE REVIEW

File: server/db/models/user.js [hunk 1]
```diff
@@ -135,12 +135,14 @@ userSchema.pre('save', async function (next) {
...
+        address: request.address,
+        state: request.state,
+        zipcode: request.zip
+      },
       async (err, res) => {
-        console.log(res);
-        request.latitude = await res[0].latitude;
-        request.longitude = await res[0].longitude;
+        request.latitude = res[0].latitude;
       }
```
Comment (leopolicastro) [inline server/db/models/user.js:146]: Why was this deleted?

**Review by leopolicastro — CHANGES_REQUESTED** [review]

File: server/db/models/user.js (hunk 1 above)
Comment (mayorgalu) [reply server/db/models/user.js:146]: @cececlar began working on this with @Juan. Got wind of the change after it was changed and reminded Juan and Cece you had set it to a specific code earlier and the app was working then. Will follow up now.

Comment (juanmartinez238) [reply server/db/models/user.js:146]: When we fetched, pulled, and ran the code from main, the usersignup form was giving us a status 400 error because it wasn't recognizing latitude so we had to change it.

**Review by leopolicastro — APPROVED** [review]
